libs/fetcher.js

```diff
@@ -223,7 +224,7 @@ function executeRequest (request, resolve, reject) {
     var service;
     try {
         service = Fetcher.getService(request.resource);
-        if (!service[op]) {
+        if ([OP_CREATE,OP_READ,OP_UPDATE,OP_DELETE].indexOf(op) < 0 || !service[op]) {
```

Comment:
@nysd Instead of moving this check to `Request` constructor and adding another try/catch, which is known to de-optimze the function for v8 engine, can we just do this check after line 431 the `if (!Fetcher.isRegistered(singleRequest.resource)) {` block, and call `next(error)` directly?  Thanks!